main.go

```diff
@@ -6,52 +6,65 @@ import (
 	"path/filepath"
 	"flag"
 
-	"github.com/docker/docker/builder/dockerignore"
-	"github.com/docker/docker/pkg/fileutils"
+	"./dockerignore"
```

Comment:
Isn't it a good practice to have all the dependencies locally for the RPM, rather than pointing to Github?